pkg/configuration/router.go

```diff
@@ -5,20 +5,60 @@
 package configuration
 
 import (
+	"bytes"
+	"context"
 	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
 	"net/http"
+	"path"
+	"strings"
+	"time"
 
 	moovhttp "github.com/moov-io/base/http"
 	"github.com/moov-io/customers/pkg/client"
+	"github.com/moov-io/customers/pkg/documents/storage"
 	"github.com/moov-io/customers/pkg/route"
+	"gocloud.dev/blob"
+	"gocloud.dev/gcerrors"
 
 	"github.com/go-kit/kit/log"
 	"github.com/gorilla/mux"
 )
 
-func RegisterRoutes(logger log.Logger, r *mux.Router, repo Repository) {
+var (
+	maxImageSize int64 = 20 * 1024 * 1024 // 20MB
+
+	// serves two purposes:
+	//   1. defines content/file types supported in the API
+	//   2. maps a supported content MIME type to it's common file extension
+	contentTypeFileExtensions = map[string]string{
```

Comment:
What's wrong with `http.DetectContentType`? In document storage I wasn't including the extension because browsers will sniff it anyway. Do we care so long as it's whitelisted?